src/sections/Careers/Careers-Programs-grid/index.js

```diff
@@ -35,14 +36,25 @@ const ProgramsGrid = ({ hide_path, sub_section }) => {
   );
 
   let path = hide_path ? "" : "Programs";
+  let programsArray = [];
+
+  const programs = data.allMdx.nodes.filter((item) => {
+    if(programsArray.indexOf(item.frontmatter.program)>=0) {
+      return false;
+    }else{
+      programsArray.push(item.frontmatter.program);
+      return true;
+    }
+  });
+
   return (
     <ProgramsPageWrapper>
       <PageHeader title="Open Source Internship Programs" subtitle="Build Your Career at Layer5" path={path} />
       <div className={sub_section ? "sub-header_wrapper" : "programs-page-wrapper"}>
         <Container>
           <div className="program-grid-wrapper">
             <Row>
-              {data.allMdx.nodes.map(({ id, frontmatter, fields }) => (
+              {programs.reverse().map(({ id, frontmatter, fields }) => (
```

Comment:
What's the use of `reverse` here? I didn't get it

Reply:
So basically what we do is fetch things in descending order of title so that I get GSoC 2021 before GSoC 2019 so that the link will always point to the latest. Once we create the list of programs with latest year, due to descending order the actual order of the programs are alphabetically descending so reverse.